src/web/module/counter/context/DataContext.tsx

```diff
@@ -1,6 +1,7 @@
 import { createContext, useContext, useState, ReactNode } from 'react';
+
 import { Counter } from '@/api/module/counter/entity';
-import { Index } from '@/api/module/counter/service';
+import { index } from '@/api/module/counter/service';
```

Comment:
More meaninful?
```suggestion
import { index as CounterService } from '@/api/module/counter/service';
```

Reply:
updated in https://github.com/RR-Ardhika/murojaah/pull/35/commits/b2171edc38efca4af40e24d6ed04a45e9dfd6276, thank you